app/controllers/albums.js

```diff
@@ -0,0 +1,29 @@
+const albumsService = require('../services/albums');
+
+function albums(req, res) {
```

Comment:
We prefer the syntax of arrow function when is possible, also use more expressive name for the controller:
```
const getAlbums = (req, res, next) => ...
```
You need a third parameter `next`, for [handling error](https://expressjs.com/en/guide/error-handling.html) mostly in controllers